DogeNetWebClient/src/app/friends/widgets/friend-card/friend-card.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="friend-wrap">
+  <mat-card class="friend-card">
+    <img
+      mat-card-image
+      src="https://material.angular.io/assets/img/examples/shiba2.jpg"
+      alt="No avatar"
+    /><mat-card-header>
+      <mat-card-title>Shiba Inu</mat-card-title>
+      <mat-card-subtitle style="margin-bottom: 0px"
+        >8 mutual friends</mat-card-subtitle
+      >
+    </mat-card-header>
+    <mat-card-actions class="actions">
+      <div>
+        <button
+          style="width: 100%; margin-bottom: 3px"
```

Comment:
Why you use inline styles instead of class?

Reply:
remade